### PR TITLE
Add a message for fixing token failure

### PR DESCRIPTION
**Pull request by yanweiguo**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -39,13 +39,14 @@ import (
...
 type contextKey string
 
-const contextKeyError = contextKey("error")
-
-const cloudPlatformScope = "https://www.googleapis.com/auth/cloud-platform"
-
-const Version = "0.1.0"
-const OSArch = runtime.GOOS + "/" + runtime.GOARCH
-const UserAgent = "cloud-run-proxy/" + Version + " (" + OSArch + ")"
+const (
```
Comment (sethvargo) [inline main.go:42]: I actually had these grouped separately on purpose. Can you revert?

File: main.go [hunk 2]
```diff
@@ -39,13 +39,14 @@ import (
...
+	contextKeyError    = contextKey("error")
+	cloudPlatformScope = "https://www.googleapis.com/auth/cloud-platform"
+	Version            = "0.1.0"
+	OSArch             = runtime.GOOS + "/" + runtime.GOARCH
+	UserAgent          = "cloud-run-proxy/" + Version + " (" + OSArch + ")"
+	ADCHintMessage     = "Try running `gcloud auth login --update-adc` first then restart the proxy."
```
Comment (sethvargo) [inline main.go:48]: I like this. I would want to preface this with "if you're trying to authenticate using gcloud, try running ..."

File: main.go (hunk 1 above)
Comment (yanweiguo) [reply main.go]: done

File: main.go (hunk 2 above)
Comment (yanweiguo) [reply main.go]: done

**Review by sethvargo — APPROVED** [review]

**Pull request merged**